src/Faction/ui/AugmentationsPage.tsx

```diff
@@ -202,6 +218,19 @@ export function AugmentationsPage({ faction }: { faction: Faction }): React.Reac
               Sort by Purchasable
             </Button>
           </Box>
+          <TextField
+            value={filterText}
+            onChange={handleFilterChange}
+            autoFocus
+            placeholder="Filter augmentations"
+            InputProps={{
+              startAdornment: <SearchIcon />,
+              spellCheck: false,
+            }}
+            style={{
+              paddingTop: "8px",
+            }}
```

Comment:
```suggestion
            sx={{ pt: 1 }}
```
Since this is a MUI component, this is preferred to directly adding padding.